src/style.css

```diff
@@ -2,0 +1,114 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, Arial, sans-serif;
+  color: var(--main-color);
+  font-weight: 300;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search--big {
+  height: 70px;
+  width: 100%;
+}
+
+.search--small {
+  height: 42px;
```

Comment:
This is a code quality issue. Similar to the `.search--big` class, you've fixed the height of the `.search--small` class. As mentioned earlier, it's generally best not to fix the height of containers unless it's a specific requirement.